provisionerd/provisionerd.go

```diff
@@ -30,11 +29,10 @@ import (
 	"github.com/coder/retry"
 )
 
-// IsMissingParameterError returns whether the error message provided
-// is a missing parameter error. This can indicate to consumers that
-// they should check parameters.
-func IsMissingParameterError(err string) bool {
-	return strings.Contains(err, runner.MissingParameterErrorText)
+// IsMissingParameterError returns whether the error is a missing parameter error.
+// This can indicate to consumers that they should check parameters.
+func IsMissingParameterError(errorCode string) bool {
```

Comment:
The name makes me feel this should accept `err error`, instead of string. Perhaps this would be better?

```suggestion
func IsMissingParameterErrorCode(code string) bool {
```